app/controllers/users_controller.rb

```diff
@@ -34,6 +34,22 @@ def update
       end
   end
 
+  def filters
```

Comment:
Add ability for it in staff based on your usecase

Reply:
done